<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="10" passed="7" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-25T18:22:08 IST" name="Suite" finished-at="2024-10-25T18:22:12 IST" duration-ms="3895">
    <groups>
    </groups>
    <test started-at="2024-10-25T18:22:08 IST" name="Test" finished-at="2024-10-25T18:22:12 IST" duration-ms="3895">
      <class name="testCases.LoginTest_01">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.LoginTest_01@4f0100a7]" started-at="2024-10-25T18:22:06 IST" name="setup" finished-at="2024-10-25T18:22:08 IST" duration-ms="2203" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="LoginTest()[pri:0, instance:testCases.LoginTest_01@4f0100a7]" started-at="2024-10-25T18:22:08 IST" name="LoginTest" finished-at="2024-10-25T18:22:09 IST" duration-ms="188" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method signature="getTitle()[pri:1, instance:testCases.LoginTest_01@4f0100a7]" started-at="2024-10-25T18:22:09 IST" name="getTitle" finished-at="2024-10-25T18:22:09 IST" duration-ms="21" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTitle -->
        <test-method is-config="true" signature="Logout()[pri:0, instance:testCases.LoginTest_01@4f0100a7]" started-at="2024-10-25T18:22:10 IST" name="Logout" finished-at="2024-10-25T18:22:12 IST" duration-ms="2151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Logout -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.LoginTest_01@4f0100a7]" started-at="2024-10-25T18:22:12 IST" name="tearDown" finished-at="2024-10-25T18:22:13 IST" duration-ms="181" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.LoginTest_01 -->
      <class name="testCases.OrderProduct_Test02">
        <test-method signature="selectDropdown()[pri:2, instance:testCases.OrderProduct_Test02@3fed2870]" started-at="2024-10-25T18:22:09 IST" name="selectDropdown" finished-at="2024-10-25T18:22:10 IST" duration-ms="1122" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- selectDropdown -->
        <test-method signature="addProductToCart()[pri:3, instance:testCases.OrderProduct_Test02@3fed2870]" started-at="2024-10-25T18:22:10 IST" name="addProductToCart" finished-at="2024-10-25T18:22:10 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addProductToCart -->
        <test-method signature="clickonCart()[pri:4, instance:testCases.OrderProduct_Test02@3fed2870]" started-at="2024-10-25T18:22:10 IST" name="clickonCart" finished-at="2024-10-25T18:22:10 IST" duration-ms="35" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickonCart -->
        <test-method signature="checkout()[pri:5, instance:testCases.OrderProduct_Test02@3fed2870]" started-at="2024-10-25T18:22:10 IST" name="checkout" finished-at="2024-10-25T18:22:10 IST" duration-ms="36" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkout -->
        <test-method signature="information()[pri:6, instance:testCases.OrderProduct_Test02@3fed2870]" started-at="2024-10-25T18:22:10 IST" name="information" finished-at="2024-10-25T18:22:10 IST" duration-ms="164" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- information -->
        <test-method signature="paymentInform()[pri:7, instance:testCases.OrderProduct_Test02@3fed2870]" started-at="2024-10-25T18:22:10 IST" name="paymentInform" finished-at="2024-10-25T18:22:10 IST" duration-ms="20" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [sauce] but found [SauceCard #31337]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [sauce] but found [SauceCard #31337]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pageObjectModel.HomePage.paymentInformation(HomePage.java:72)
at testCases.OrderProduct_Test02.paymentInform(OrderProduct_Test02.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- paymentInform -->
        <test-method signature="finish()[pri:8, instance:testCases.OrderProduct_Test02@3fed2870]" started-at="2024-10-25T18:22:10 IST" name="finish" finished-at="2024-10-25T18:22:10 IST" duration-ms="35" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finish -->
      </class> <!-- testCases.OrderProduct_Test02 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
